packages/client/src/utils/gateway.ts

```diff
@@ -32,6 +32,7 @@ export type Scalars = {
   Int: number
   Float: number
   Date: any
+  FieldValue: any
```

Comment:
Is there a way to tell the codegen what the scalar types are? :thinking:

Reply:
Wer actually started with a 

```
type FieldValue {
  valueString: String
  valueInteger: Int
  valueBoolean: Boolean
}
```

type 😄 It just turned out to be so complicated to do the type conversions and in some cases we even had to implement some not-so-ideal heuristics. We could also need to have a field for Float vs Integer and so on.